lib/open_project/translations/patches/redmine_i18n_patch.rb

```diff
@@ -22,7 +21,7 @@ def all_languages_with_translation_plugin
       end
 
       # do not count javascript translations as separate langauges
-      plugin_languages.reject! {|l| l.to_s[0..2] == 'js-' }
+      plugin_languages.reject! { |l| l.to_s[0..2] == 'js-' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.